### DISCUSSION TITLE
Set up Github Sponsor?

### DISCUSSION THREAD

**Discussion by tausifcreates** [Totodore/socketioxide#71] [Ideas]
I am sure some people will subscribe to support the development of socketioxide.

**Comment by Totodore** [comment]
It is a good idea, I'm going to do it. But I don't expect very much at the moment.

**Reply by tausifcreates** [reply]
I subbed 😄 Just for 1$ tho. I hope I will be able to support more in future.

**Comment by tausifcreates** [comment]


**Reply by Totodore** [reply]
sure

**Comment by tausifcreates** [comment]
The sponsor link takes nowhere :(

**Reply by Totodore** [reply]
It will be ok once my profile is validated

**Comment by tausifcreates** [comment]
Creating tiers for the sponsors is a good idea. People who visit the sponsor page generally become confused on how much they should subscribe for. so a few tiers (1$, 3$, 5$, 10$ etc) with perks ( Discord membership, One-2-One mentoring) will be a nice addition, like Patreon.

Maybe down the road you can offer commercial cloud services with Socketio. For now, some marketing is necessary. We know how popular of a tool Socketio is and rust can ramp up socketio's performance by large factors.

**Reply by Totodore** [reply]
I agree with you but for the moment socketioxide is absolutely not production ready so I won't propose any one comercial support or mentoring. Also I currently don't want to manage any  community/marketing stuff.

**Reply by tausifcreates** [reply]
Fair. Even if you do not offer perks, tiers are great for avoiding confusions and excite visitors (As I see how others arrange the sponsor page).
![image](https://github.com/Totodore/socketioxide/assets/93582436/4aba253b-5d3e-485e-a907-95771993a96f)
**Lastly**, leave the marketing duty on the community (laughing)
